contracts/0.8.25/vaults/dashboard/Dashboard.sol

```diff
@@ -338,7 +338,7 @@ contract Dashboard is NodeOperatorFee {
 
     /**
      * @notice Mints stETH tokens backed by the vault to the recipient.
-     * !NB: this will revert with `VaultHub.ZeroArgument("_amountOfShares")` if the amount of stETH is less than 1 share
```

Comment:
Why remove `VaultHub.` prefix? Dashboard also has a ZeroArgument error, may clash.

Reply:
Because errors does not depend on the contract that revert them. So it would be the same signature if it's reverts in VaultHub or somewhere else.